### PR TITLE
Filter out easy alerts and updated the Falco_rules

### PR DESCRIPTION
**Pull request by raviranjanelastisys**
Signed-off-by: Ravi Ranjan <ravi.ranjan@elastisys.com>

**What type of PR is this?**

 /kind feature

**Any specific area of the project related to this PR?**

 /area falco-chart

**What this PR does / why we need it**:

Bases on our investigations and finding , updated the rules to `write_etc_common` , `Launch Package Management Process in Container` , `falco_privileged_images` & `falco_sensitive_mount_containers`.

**Which issue(s) this PR fixes**:
N/A

Fixes #

**Special notes for your reviewer**:

This issue consists in filtering a few "obvious" false-positives.
Updated the rules to `write_etc_common` , `Launch Package Management Process in Container` , `falco_privileged_images` & `falco_sensitive_mount_containers`.

**Checklist**

- [X] Chart Version bumped
- [ ] Variables are documented in the README.md
- [X] CHANGELOG.md updated

### CODE REVIEW

**Review by cristiklein — CHANGES_REQUESTED** [review]
Please fix the minor issue I raised. Otherwise looks good to me.

File: falco/rules/falco_rules.yaml [hunk 1]
```diff
@@ -1467,7 +1468,7 @@
   tags: [filesystem, software_mgmt, mitre_persistence]
 
 - macro: postgres_running_wal_e
-  condition: (proc.pname=postgres and proc.cmdline startswith "sh -c envdir /etc/wal-e.d/env /usr/local/bin/wal-e")
+  condition: (proc.pname=postgres and proc.cmdline startswith "sh -c envdir \"/run/etc/wal-e.d/env\" wal-g wal-push")
```
[the remark this answers is not in the document]
Comment (raviranjanelastisys) [reply falco/rules/falco_rules.yaml:1471]: @cristiklein Thanks Cristian !! As suggested , this is already opened PR in this commit - 
https://github.com/falcosecurity/charts/pull/319/files#diff-9e83f5b3c5b980da0d38329ea563be1e28722c0f04da90000bd19f3f6160cb97R1470

[the remark this answers is not in the document]
Comment (raviranjanelastisys) [reply falco/rules/falco_rules.yaml:1471]: @cristiklein Is that you suggest ?

`condition: (proc.pname=postgres and proc.cmdline startswith "sh -c envdir /etc/wal-e.d/env /usr/local/bin/wal-e") or (proc.pname=postgres and proc.cmdline startswith "sh -c envdir \"/run/etc/wal-e.d/env\" wal-g wal-push")`  ??

[the remark this answers is not in the document]
Comment (cristiklein) [reply falco/rules/falco_rules.yaml:1471]: Exactly! Maybe with a few strategic newlines to make it more readable. :smile:

[the remark this answers is not in the document]
Comment (raviranjanelastisys) [reply falco/rules/falco_rules.yaml:1471]: @cristiklein Thanks I have done the changes in the readable format .

`condition: ((proc.pname=postgres and proc.cmdline startswith "sh -c envdir /etc/wal-e.d/env /usr/local/bin/wal-e") or
                   (proc.pname=postgres and proc.cmdline startswith "sh -c envdir \"/run/etc/wal-e.d/env\" wal-g wal-push"))`

**Review by leogr — CHANGES_REQUESTED** [review]
Hey @raviranjanelastisys 

You should propose changes to rules directly in the [falco repository](https://github.com/falcosecurity/falco/tree/master/rules).

The chart just uses a copy of those files, so we cannot modify them here.

Feel free to ping me if you need any help.

cc @cristiklein

**Comment by raviranjanelastisys** [comment]
> so we cannot modify them here.

Thanks @leogr  !! I will opened PR in the suggested repository soon and let you know :) ...

**Comment by leogr** [comment]
> > so we cannot modify them here.
> 
> Thanks @leogr !! I will opened PR in the suggested repository soon and let you know :) ...

Great, thank you! We really appreciate your contribution :+1: 

So I'm closing this PR in favor of the one you will open in the falco repository.

/close

**Comment by raviranjanelastisys** [comment]
Opened the PR against the correct repos as suggested - https://github.com/falcosecurity/falco/pull/1981

**Pull request closed without merging**